openlibrary/core/lists/model.py

```diff
@@ -376,7 +376,7 @@ def _load_solrdata(self):
                 return {
                     'ebook_count': sum(doc.get('ebook_count_i', 0) for doc in result.docs),
                     'edition_count': sum(doc.get('edition_count', 0) for doc in result.docs),
```

Comment:
`ebook_count` and `edition_count` are going to be limited to the ebooks and editions of the solr results limit (which seems to be 10). This is the existing code.
There is a way to get total ebook count outside of solr, 
[subject pages](https://dev.openlibrary.org/subjects/person:elizabeth_i_queen_of_england_(1533-1603)) do it, but it might be expensive on larger lists?